src/main/java/Task.java

```diff
@@ -0,0 +1,24 @@
+public class Task {
+    private String task;
```

Comment:
You could consider renaming "task" to something like "description" or "taskDescription" for better meaning